js/logo.js

```diff
@@ -6357,7 +6357,24 @@ function Logo() {
 
         this.pickupPoint[turtle] = null;
     };
+    
+    /**
+     * Adds a repeat notation.
+     * @privileged
+     * @param   turtle
+     * @param   repeatCount
+     * @param   repeatedNotes
+     * @returns {void}
+     */
+    var repeatCount=this._doBreak.queuelength;
+    var repeatedNotes=this.turtles.turtleList[turtle].queue.pop();
```

Comment:
And this?

Reply:
in doBreak function,i think the queueLength  is same as the no. of count for repeat block.And for repeatedNotes,I am not that sure.

Reply:
@walterbender  Is it right or not?